src/styles/foundation/_focus-ring.scss

```diff
@@ -2,8 +2,8 @@
 /// @param {String} $size - The size of the border radius on the focus ring.
 ///
 
-@mixin focus-ring($size: 'base') {
-  $offset: rem(1px);
+@mixin focus-ring($size: 'base', $offset: rem(1px), $show: 0) {
```

Comment:
Needed to add the $offset, because when the element has a border, the `:after` positioning of `top left` changes by the width of the border. This offset will the width of the border + the 1 px gap.